from distutils.log import error


a = 1

raise Exception